gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #224: Can't Logout

### Description

* Made the "log out" button log you out immediately instead of asking for confirmation

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1. Log in
1. Click "Log out (name)" and see that you're immediately taken to the home page and you're not asked to confirm that you want to log out
![log out](https://user-images.githubusercontent.com/13837978/70299072-d05b6900-17c1-11ea-864f-b803aba05a77.gif)


### Miscellaneous Comments

* I decided not to follow [this answer](https://stackoverflow.com/a/18135023) because [this other answer](https://stackoverflow.com/a/14587231) discouraged it. That's because modern browsers prefetch links so if all it takes is a GET request to log you out, your browser might follow that link and log you out when you don't want to. So I thought that was pretty funny.

### Checklist
